activerecord/test/cases/adapters/postgresql/legacy_migration_test.rb

```diff
@@ -0,0 +1,49 @@
+require "cases/helper"
+
+class PostgresqlLegacyMigrationTest < ActiveRecord::PostgreSQLTestCase
+  class GenerateTableWithoutBigserial < ActiveRecord::Migration::Compatibility::V5_0
+    def change
+      create_table :legacy_integer_pk do |table|
```

Comment:
@kamipo I've modified the `create_table` method to ensure the ID columns auto-increment. Added tests for both adapters as well.